dist/index.html

```diff
@@ -1,13 +1,110 @@
+        **/ -->
+        <div class="w-full">
+            <div>
+                <div class="flex gap-3 justify-end items-center px-6 h-10v">
+                    <a href="#" class="text-gray-500 cursor-pointer hover-underline  text-sm">Gmail</a>
+                    <a href="#" class="text-gray-500 cursor-pointer hover-underline text-sm">Images</a>
+                    <svg class="h-10 hover:rounded-full hover: cursor-pointer hover:bg-gray-200 p-2" focusable="false"
+                        viewBox="0 0 24 24">
+                        <path
+                            d="M6,8c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM12,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM6,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM6,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM12,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM16,6c0,1.1 0.9,2 2,2s2,-0.9 2,-2 -0.9,-2 -2,-2 -2,0.9 -2,2zM12,8c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM18,14c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2zM18,20c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2 -2,0.9 -2,2 0.9,2 2,2z">
+                        </path>
+                    </svg>
+                    <div class=" hover:rounded-full hover:bg-gray-200 p-1">
+                        <img class="rounded-full h-6 cursor-pointer"
+                            src="https://lh3.googleusercontent.com/ogw/AOh-ky0MfVrW-_Q9U_1JG520JfRIJ0N-UIalZyWTP37LBg=s64-c-mo"
+                            alt="img">
+                    </div>
+
+                </div>
+            </div>
+        </div>
+        <!-- /**
+            ! First Divisison is here    
+            **/ -->
+        <div class="w-full flex-1">
+            <!-- /**
+            ! Google Image is placed here
+            **/ -->
+            <div class="flex sm:h-22v justify-center items-center">
+                <img class="sm:h-5rem h-3rem mt-6 sm:mt-0"
+                    src="https://www.google.com/images/branding/googlelogo/2x/googlelogo_color_272x92dp.png" alt="google">
+            </div>
+            <!-- 
+                !Google input and all the Icons 
+                -->
+            <div>
+                <div class="h-10 flex justify-center items-center relative">
+                    <svg class="h-4 absolute left-30% text-gray-800" xmlns="http://www.w3.org/2000/svg" fill="none"
+                        viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6">
+                        <path strokeLinecap="round" strokeLinejoin="round"
+                            d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z" />
+                    </svg>
+
+                    <input type="text"
+                        class="outline-none border rounded-full px-16 sm:px-12 border-gray-300 hover:shadow-lg  sm:h-11 w-43%">
+                    <img class="h-6 absolute right-30% " src="./image/google_Icon-removebg-preview.png" alt="">
```

Comment:
right-30% is customized css

Reply:
No this is cutom css, I named it 30% if you check well it is not in bracket[